src/Components/test/testassets/BasicTestApp/PrependMessageLoggerProvider.cs

```diff
@@ -9,6 +9,7 @@
 
 namespace BasicTestApp
 {
+    [ProviderAlias("PrependMessage")]
```

Comment:
What does this do?

Reply:
It allows us to use "PrependMessage" in the configuration file to configure this particular provider instead of having to use the fully qualified name.